pyramid_assemble.py

```diff
@@ -46,7 +49,9 @@ def format_shape(shape):
 
 def construct_xml(filename, shapes, num_channels, dtype, pixel_size=1):
     img_uuid = uuid.uuid4().urn
-    if dtype == np.uint16:
+    if dtype == np.uint32:
```

Comment:
I don't think uint32 intensity images will ever be a thing (I believe skimage is actually going to deprecate them or completely drop support) so we can just error if anyone tries that. How about this `if` block gets moved up to `main`, and the `uint32` branch can then error out if `is_mask` is True. Then `construct_xml` can take `ome_dtype` directly instead of `dtype`. I think that covers everything?

Reply:
I assume you meant error out if `is_mask` is `False` while dtype is `uint32`. I've moved the conditional to the main function and only allow an `ome_dtype` of `uint32` when `is_mask` is `True`.